src/openrct2/actions/LandBuyRightsAction.cpp

```diff
@@ -163,7 +164,7 @@ GameActions::Result LandBuyRightsAction::MapBuyLandRightsForTile(const CoordsXY&
                 uint16_t baseZ = surfaceElement->GetBaseZ();
                 MapInvalidateTile({ loc, baseZ, baseZ + 16 });
             }
-            res.Cost = gConstructionRightsPrice;
+            res.Cost = OpenRCT2::GetGameState().ConstructionRightsPrice;
```

Comment:
Can you put a `using namespace OpenRCT2;` under the include list and drop the `OpenRCT2::` prefix? (This applies to any occurrence outside `.h` files)